src/App.js

```diff
@@ -20,7 +22,9 @@ function App() {
         <Nav history={history}></Nav>
         <Switch>
           <Route exact path='/' component={Home}/>
+          <Route exact path='/schedule' component={Schedule}/>
           <Route exact path='/resources' component={Resources} />
+          <Route exact path='/workshops' component={Workshops}/>
```

Comment:
Why is workshops being added here? Didn't we already include that

Reply:
Oh yeah we already did, I don't know why I added it. Do I need to delete it?

Reply:
This is a guess, but it might be because I pulled the latest changes from master onto the branch I worked with for adding the schedule.